Formula/tree-sitter.rb

```diff
@@ -88,5 +99,14 @@ def install
     EOS
     system ENV.cc, "test_program.c", "-L#{lib}", "-ltree-sitter", "-o", "test_program"
     assert_equal "tree creation failed", shell_output("./test_program")
+
+    # test `tree-sitter web-ui`
+    ENV.delete "CPATH"
+    system bin/"tree-sitter", "build-wasm"
+    fork do
+      exec bin/"tree-sitter", "web-ui"
+    end
+    sleep 10
+    system "killall", "tree-sitter"
```

Comment:
I'm unsure about these tests. They seem to only be testing the tree-sitter CLI. In particular, I can run them even without having installed the WebAssembly binding you added to the `install` block.

Reply:
The result of this test will open browser and see an editor like [tree-sitter playground](https://tree-sitter.github.io/tree-sitter/playground).

Reply:
I understand, thanks!